src/app/toollbar/toollbar.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
There is a typo in the file name for all 4 files of this component.
It will be toolbar/toolbar.component.ts
Single 'l' and not a double 'll'.

Reply:
Will take care of it.